gulpfile.js

```diff
@@ -33,4 +33,17 @@ elixir(function(mix) {
         'socialmediashare.js'
     ], 'public/js/scripts.js', 'public/js');
 
+    /**
+     * Compile sass files to css
+     */
+    mix.sass('main.scss')
+       .sass('audioplayer.scss');
```

Comment:
Here, we have the sass code, does it mean we now use sass because I see no reference of it in the blade files.

Or is it just the minified styles.css we now use? @andela-jkariuki